livechat/overview/get_started.rst

```diff
@@ -0,0 +1,83 @@
+==========================
+Get Started with Live Chat
+==========================
+
+Real-time chat is a cost-effective tool and a great way to bring visitors back to your website. It
+is an accessible and convenient customer support channel, and can increase efficiency, considering
+that this way, support representatives can handle more than one case at a time.
```

Comment:
This whole sentence doesn't work. It doesn't flow and it doesn't speak to the main benefits of live chat (human interactions, fast response time...)